### DISCUSSION TITLE
Add Nelua to Kostya benchmarks

### DISCUSSION THREAD

**Discussion by dumblob** [edubart/nelua-lang#34] [Ideas]
Despite nelua being this young, I'd suggest to submit a PR to https://github.com/kostya/benchmarks/ to get a glimpse of how it might behave and to attract audience (testers, devs).

**Comment by edubart** [comment]
A while ago I've ported some benchmarks from there in https://github.com/edubart/benchmarks however I will wait a beta release of Nelua to make a PR as of right know I don't want to attract many testers and devs without a proper beta release. The benchmarks results if you are curious are very close to C results on my machine.
